Dockerfile

```diff
@@ -2,6 +2,8 @@ from ruby:2.5.1
 
 RUN apt-get update -qq && apt-get install -y build-essential libpq-dev curl
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
+RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
```

Comment:
This installs chrome, which is what will act as the web framework the tests use.